content/customize-gnome.md

```diff
@@ -18,117 +18,210 @@ section: software
 
-## Gnomenu
+<details>
+  <summary>Click to expand</summary>
 
-[Gnomenu](https://extensions.gnome.org/extension/608/gnomenu/)
+Since Desktop icons have been deprecated in mainline GNOME, Pop!_OS includes a GNOME extension to bring this feature back.
 
-![gnomenu](/images/customize-gnome/gnomenu.png)
+</details>
 
-This extension adds a traditionally styled, full featured menu to GNOME.  This menu is similar to a traditional Window's Start Menu, and is highly customizable for how it looks and functions.
+### [Pop COSMIC](https://github.com/pop-os/cosmic)
 
----
+<details>
+  <summary>Click to expand</summary>
 
-## Appindicator Support and TopIcons Plus
+![cosmic-dock](/images/customize-gnome/cosmic-dock.png)
 
-[Appindicator Support](https://extensions.gnome.org/extension/615/appindicator-support/)
-[Top Icons](https://extensions.gnome.org/extension/1031/topicons/)
+Computer Operating System Main Interface Components **(C.O.S.M.I.C.)**
 
-![topicons](/images/customize-gnome/topicons.png)
+COSMIC is the name for the main shell components in Pop_Shell (separate from the tiling and window-management components). It makes the following changes to the standard GNOME Shell environment:
 
-Appindicator Support and TopIcons Plus both serve to add the ability for programs to use an indicator icon to show they are running.  This functionality has been removed in recent versions of GNOME, and an extension is needed for the functionality.  These 2 programs work slightly differently, and work better in some setups vs. others.  If one doesn't work, try the other one.
+- Dock
+- Multi-Monitor improvements
+- Configuration options
+- Separated Workspaces overview from Applications.
 
-See [blogs.gnome.org](https://blogs.gnome.org/aday/2017/08/31/status-icons-and-gnome/) for more background on the removal of the support from default GNOME environment.
+Most components can be configured to fit the user's workflow and preferences; presets are included for both keyboard-focused and mouse-focused navigation and use.
 
----
+</details>
 
-## Internet Radio
+### [Pop Shell](https://github.com/pop-os/shell)
 
-[Internet Radio](https://extensions.gnome.org/extension/836/internet-radio/)
+<details>
+  <summary>Click to expand</summary>
 
-![internet-radio](/images/customize-gnome/internet-radio.png)
+![pop-shell](/images/customize-gnome/pop-shell.png)
 
-Internet Radio adds a quick menu integration to play various radio stations found online.  A few default stations are included, and many more can be found at [www.internet-radio.com](https://www.internet-radio.com/).  Simply right-click on the `.m3u` file and choose 'Copy link address', and add that as a station to the Internet Radio icon at the top right of the menu bar.
+An extension designed to add `i3wm`-style auto-tiling and related keyboard navigation features to the GNOME Desktop.
 
----
+For details refer to the Github page (linked above) or view the tutorial content [here](/articles/pop-basics)
```

Comment:
Missing period at the end of this sentence

Reply:
Also the product of being asked to handle links in two different ways. I know there should be a period. I've started adding them back where I see them.

Thanks for catching this.

Reply:
Fixed.